### ISSUE TITLE
[interface] wastes lots of space

### ISSUE THREAD

**Issue by h-2** [wireapp/wire-android#297]
I know that design issues are kind of subjective and that wire is not aiming for material design, but I really think that it should be able to display more than 1.5 lines of dialog on one screen :frowning: 

![screenshot_20161108-081300](https://cloud.githubusercontent.com/assets/2996863/20113517/ff4f062c-a5f0-11e6-8675-f7098e64ed59.png)

I have shown where a lot of space is wasted. The most important thing is the additional time notifier in the middle, which seems to always be inserted after the conversation was interrupted for a while.

Some more humble suggestions:
  * move attachments, gifs and ping buttons behind one button to save the extra line
  * move name above or below avatar to have more space for text
  * remove redundancy between name in the top and avatar in the chat

Also I think it would vastly improve the usability, if the interface (input and topbar) would be more distinct from scrollview / chat area.

Anyways, all criticism is constructive. Thanks for the app!

**Comment by bjoernherbig** [comment]
thanks. 
we take it as constructive feedback and food for thoughts. we're constantly discussing improvements to the ui so your feedback is appreciated. 
i'll close this for now to keep the issues lean, but we're taking your feedback into the discussion.

**Comment by ElucGeek** [comment]
I feel exactly the same as @h-2 and was about to open an issue for this. In one to one conversation you can just see the last post with a couple of line, then it's hidden. When replaying in a group chat with many activity it's hard to follow while typing, you have to stop typing, close the keyboard, scroll, read, and come back. The more space you can save vertical, would be the better IMHO.
Is there any update about this ?
